source/changelog.txt

```diff
@@ -12,6 +12,71 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.2.1
+------
+
+*Released March 19, 2024*
+
+Fixes a bug in 2.2.0 where ``require('<module>')`` caused an error in 
+script mode:
+
+- :issue:`MONGOSH-1738` - ``require`` not working in script mode.
+
+v2.2.0
+------
+
+*Released March 11, 2024*
+
+.. important::
+     This release is affected by a bug, fixed in 2.2.1, where ``require('<module>')`` 
+     caused an error in script mode.
```

Comment:
I don't see a build error for this and the rendering seems ballpark right, so maybe Snooty is making some adjustments, but the indentation looks wrong.

There should be a blank line after the directive.   Lines under the directive should have three space indents to bring them in line with the first letter of the directive type.